frontend/src/old-pages/CustomImages/CustomImages.tsx

```diff
@@ -43,22 +43,18 @@ const imageBuildPath = ['app', 'customImages', 'imageBuild'];
 const selectCustomImagesList = (state: any) => state.customImages.list;
 
 function CustomImagesList() {
-  const images = useSelector(selectCustomImagesList) || [];
-  const open = useState([...imageBuildPath, 'dialog']);
+  const images: ImageInfoSummary[] = useSelector(selectCustomImagesList) || [];
```

Comment:
FYI: I slight better way to do this would be to correctly type your selector. (The ideal way would be to type the Store itself, but we'll get there eventually)

The reason why it is slightly better to let TS infer the type for your `const images` is that if you were to change or inadvertently break the selector, it would yell at you in the rest of the code :) 

```ts
const selectCustomImagesList = (state: any) : ImageInfoSummary[] => state.customImages.list;
//...
const images = useSelector(selectCustomImagesList) || [];
```